apps/dotcom/client/src/tla/components/TlaFileError/TlaFileError.tsx

```diff
@@ -35,15 +35,15 @@ export function TlaFileError({ error }: { error: unknown }) {
 		case TLSyncErrorCloseEventReason.NOT_AUTHENTICATED: {
 			return (
 				<TlaFileErrorContent
-					header="Private file"
-					para1="Contact the owner to request access."
+					header="Sign in"
+					para1="You are currently logged out and need sign in to view this file."
```

Comment:
this statement will be false most of the time, and it will hurt UX a little. I don't think there's a security risk in keeping it how it was. What information does it give a 'snooper' that they don't already have? 'this URL points to a tldraw file' will already be known to them.

Reply:
also 'you are currently logged out' is redundant. Just say 'you need to sign in'

Reply:
> this statement will be false most of the time

Why would the statement be false most of the time? The majority of these cases will be:
- the owner is logged out, they have the link because they've bookmarked it, they need to sign in
The other case where a file was shared but then made private is honestly more rare. We can't confidently say you need to contact the owner if a person is logged out, we just don't have enough information. But all that is moot in my mind because we will add per-email invites sooner than later.

The current copy is confusing, esp. if you're the owner: "Contact the owner" when the person reading it is the owner, and then having a CTA to sign in are sending mixed messages. There's really no way to contact the owner at the moment (although we should add that, to 'ring the doorbell').

> What information does it give a 'snooper' that they don't already have? 'this URL points to a tldraw file' will already be known to them.

Again, it's slight but it's there. If someone comes across this link (say, it was posted publicly inadvertently, in a future where we have per-email invites), the person coming across this link could discover that this is either a truly private file they've come across or a file that is shared across multiple people. They could then infer from other signals online, say if it's shared, who possibly the other invitees are and can try to phish for access from one of those invitees.

Reply:
> Again, it's slight but it's there. If someone comes across this link (say, it was posted publicly inadvertently, in a future where we have per-email invites), the person coming across this link could discover that this is either a truly private file they've come across or a file that is shared across multiple people. They could then infer from other signals online, say if it's shared, who possibly the other invitees are and can try to phish for access from one of those invitees.

how could they make that distinction though? i.e. 'truly private' vs 'selectively shared' - that's not exposed. All they know is that it exists and they don't have access.

I'm gonna relent here anyway. I think your suggestions create slightly worse UX for the (imo more common) use case where it's a guest trying to visit the file, but it's not a huge difference and I don't think it's worth spending all this time fussing over.